apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -96,6 +96,19 @@ static void ble_device_scanned_cb_with_filter(ble_scanned_device *scanned_device
 	}
 }
 
+static void ble_device_scanned_cb_with_filter(ble_scanned_device *scanned_device)
+{
+	RMC_LOG(RMC_CLIENT_TAG, "Find RMC with scan filter!!!\n");
+	printf("scanned mac : %02x:%02x:%02x:%02x:%02x:%02x\n", 
+		scanned_device->addr.mac[0],
```

Comment:
why did you separate log? RMC_LOG and printf

Reply:
Okay, I will modify this in next PR.